apps/backend/config/app_config.ts

```diff
@@ -51,9 +51,54 @@ export default class AppConfig {
   }
 
   getSSLConfig() {
+    let sslKey, sslCert, sslCA;
```

Comment:
maybe we should gate looking at all of these values on if the DATABASE_SSL is true.  if the user has garbage text in there, i don't think it matters since we say in the comments of the env-example that we only look at them if the other variable is true.  use-case i'm envisioning is someone leaving a valid sslkey/cert/ca in place but changing database_ssl to false as they do some debugging.